### PR TITLE
add yannicks deployment config repo

### PR DESCRIPTION
**Pull request by yannickholzenkamp**
Am I doing this right, @niiku ? 😅

### CODE REVIEW

File: apps/okd4-yannickholzenkamp-apps.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#  - groupName: "team1"
+#    clusterRoleName: "admin"
+
+clusterResourceWhitelist:
+
+namespaceResourceBlacklist:
+#  - group: 'apps'
+#    kind: 'DaemonSet'
+
+applications:
```
Comment (christiansiegel) [inline apps/okd4-yannickholzenkamp-apps.yaml:24]:
```suggestion
applications: {}
```

File: bootstrap/values.yaml [hunk 2]
```diff
@@ -12,3 +12,4 @@ bootstrap:
  - name: okd4-bal-code-camp-cat-apps
  - name: okd4-this-or-that-apps
  - name: sven-goldcard-spring-boot-rest-demo-config
+ - name: okd4-yannickholzenkamp-apps.yaml
```
Comment (christiansiegel) [inline bootstrap/values.yaml:15]:
```suggestion
 - name: okd4-yannickholzenkamp-apps
```

**Review by christiansiegel — CHANGES_REQUESTED** [review]
almost :wink:

**Review by christiansiegel — APPROVED** [review]
lgtm

**Comment by yannickholzenkamp** [comment]
Thanks!

**Comment by yannickholzenkamp** [comment]
Also added two applications in my apps.yaml. @christiansiegel, could you merge it if it's ok? I don't have the permissions to do that 🙂

**Comment by christiansiegel** [comment]
You can also add those two webhooks (both with json content type) to your config repo for the future:
- https://tekton-syncapps-route-tekton-chatopshandler.apps.okd.baloise.dev/
- https://argocd.baloise.dev/api/webhook

The first will sync the applications (creates one application entry in the yaml here for every folder in your repo). The second triggers ArgoCD to sync the config to the cluster.

**Comment by yannickholzenkamp** [comment]
Will do that - thanks!

**Comment by yannickholzenkamp** [comment]
@christiansiegel do you have an idea what i missed when my apps are still not showing up in argocd?

**Comment by christiansiegel** [comment]
> @christiansiegel do you have an idea what i missed when my apps are still not showing up in argocd?

Looks like ArgoCD is stuck trying to sync the cluster infra apps since last night. @niiku could you please have a look at this?

**Comment by niiku** [comment]
The OKDv4 update went a bit south but all should be working again. DockerHub has now pull limits...

**Comment by yannickholzenkamp** [comment]
Thanks for fixing, the applications are now visible in ArgoCD. Somehow the resources are missing / the images are not deployed. Do you maybe have a hint for me what's wrong w/ my configuration?

**Comment by niiku** [comment]
@yannickholzenkamp Rename Charts.yaml to Chart.yaml so ArgoCD recognizes it as Helm Chart

**Comment by yannickholzenkamp** [comment]
Ok, stupid me 🤦 Thanks a lot!

**Pull request merged**